Code/Framework/AzToolsFramework/AzToolsFramework/Entity/EditorEntityActionComponent.cpp

```diff
@@ -62,6 +62,10 @@ namespace AzToolsFramework
             bool AreComponentRequiredServicesMet(const AZ::Component* component, const AZ::Entity::ComponentArrayType& existingComponents)
             {
                 auto componentDescriptor = GetComponentDescriptor(component);
+                if (!componentDescriptor)
+                {
+                    return false;
+                }
```

Comment:
This is a case where returning false here is absolutely the correct thing to do... however, we shouldn't really get into this situation to begin with (Right?).   It means that this component, the one being activated, and is checking for its required stuff, cannot itself be found in the list of all components that are registered.

Its a sign of a much bigger issue, maybe its worth an AZ_Assert, or at least an an AZ_Error, - the only reason I suggest maybe an assert, is that you really want to not be here in the first place...  It could be something harmless like this is a component that no longer exists in the codebase, but was deserialized (So its registered to the serializer but not the component descriptor system) but it could be a lot more serious, like, some sort of dead stripping has erased the component descriptor and weird things are going to happen later without you knowing that this is the problem...

Its quite possible the real error began here, if you look at the log
```
Prefab: Converting to runtime Prefab 'PlayerSpawner' failed, Error: Entity 'Player Spawner 1' [13453922473772743748] - export entity failed. Error: Entity 'Player Spawner 1' [13453922473772743748] - dependency evaluation failed. Error: No descriptor registered for Component class 'NetworkPlayerSpawnerComponent'. .
```

It was trying to convert the `Player Spawner 1` entity from an editor entity to a game runtime entity, and failed to do so properly.  Its unclear why the NetworkPlayerSpawnerComponent has no descriptor in the Editor, perhaps it lives in a DLL which only gets loaded by the server?

Nevertheless, I think this is a good change - we should not derefrence these pointers without checking first especially since the data comes from serialization.  A further investigation should be done (or a new ticket filed) to investigate the above warning / error.